components/atom/popover/README.md

```diff
@@ -33,5 +33,25 @@ import AtomPopover, { atomPopoverPositions } from '@s-ui/react-atom-popover'
 
 ```
 
+Some times you may need to programmatically recalculate the `AtomPopover` position in order to fix visual inconsistencies (i.e. a change on the popover content could change the popover size, and therefore recalculating the position may be needed to ensure that popover is displayed as expected). To do so, it's possible to provide a component in the `AtomPopover`'s `content` property, this component will receive a function inside a property named `update`, calling this function will result on a popover's position recalculation.
```

Comment:
Add one of those to the documentation plz
https://popper.js.org/docs/v2/lifecycle/#manual-update
https://popper.js.org/docs/v1/#Popper.scheduleUpdate